scripts/semconv/templates/semantic_metrics.j2

```diff
@@ -1,40 +1,121 @@
-# Copyright The OpenTelemetry Authors
```

Comment:
Nit: Do we not need the OpenTelemetry copyrights for template files?

Reply:
great catch, added